tests/Node.Tests.ps1

```diff
@@ -6,11 +6,17 @@ param (
 Import-Module (Join-Path $PSScriptRoot "../helpers/pester-extensions.psm1")
 
 function Get-UseNodeLogs {
-    $logsFolderPath = Join-Path -Path $env:AGENT_HOMEDIRECTORY -ChildPath "_diag" | Join-Path -ChildPath "pages"
+    $homeDir = $env:HOME
+    if ([string]::IsNullOrEmpty($homeDir)) {
+        # GitHub Windows images don't have `HOME` variable
+        $homeDir = $env:HOMEDRIVE
+    }
+
+    $logsFolderPath = Join-Path -Path $homeDir -ChildPath "runners/*/_diag/pages" -Resolve
```

Comment:
Is there any variable for agent folder?
`runners` can contain more than 1 agent

Reply:
Currently we have two folders in the `runners` dir: `2.272.0` and `2.273.0`. Only `2.272.0` runner has the `pages` folder in the `_diag` dir.